src/main/java/com/faforever/client/fa/ForgedAllianceService.java

```diff
@@ -52,7 +52,8 @@ public Process startGame(int uid, @Nullable Faction faction, @Nullable List<Stri
                            String ratingType, int gpgPort, int localReplayPort, boolean rehost, Player currentPlayer) throws IOException {
     Path executable = getExecutable();
 
-    Optional<LeaderboardRating> leaderboardRating = Optional.ofNullable(currentPlayer.getLeaderboardRatings().get(ratingType));
+    Optional<LeaderboardRating> leaderboardRating = Optional.of(currentPlayer.getLeaderboardRatings())
+        .map(rating -> rating.get(ratingType));
```

Comment:
Should we use the RatingUtil here as well?

Reply:
Possibly although currently RatingUtil just returns the shown rating so mean-3*deviation while this needs the mean and deviation seperately

Reply:
:+1: